### PR TITLE
Feature/add filter

### PR DESCRIPTION
**Pull request by jeremiahblackol**
# PR Description
Please include a summary of the change:
Adds filter dropdown to dom.

Commit Emojis:

:tada: Initial commit
:construction: HTML modifications/updates.
:art: CSS modifications/updates.
:crystal_ball: JS modifications/updates.
:lipstick: hotfix any code.
:bar_chart: new tests
:camera_flash: adding screenshots
:bento: Adding new assets or images
:beetle: If it is a bug or issue fix, describe the issue that was fixed.

If it is a new feature, describe the feature and the iteration of the project spec it addresses.
this addresses iteration 1

Note any questions or struggles as well.

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist:
- [ ] My code follows the Turing style guidelines
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -91,32 +93,11 @@ function viewFavorites() {
   } else {
     favButton.innerHTML = 'Refresh Favorites'
     cardArea.innerHTML = '';
-    user.favoriteRecipes.forEach(recipe => {
```
Comment (Melizzo) [inline src/scripts.js:94]: Boom - this is great! way to consolidate the function - this is going to be great for DomUpdates!

File: src/scripts.js [hunk 2]
```diff
@@ -258,54 +217,84 @@ function displayDirections(event) {
   })
 }
 
-function getFavorites() {
```
Comment (Melizzo) [inline src/scripts.js:261]: Yeah - eliminated an unnecessary function!

File: src/index.html [hunk 3]
```diff
@@ -16,6 +16,65 @@ <h1>What's Cookin'</h1>
       <label for='view-recipes-to-cook' class='hidden'>View Recipes To Cook</label>
       <button id='view-recipes-to-cook' class='view-recipes-to-cook navButton'>View Recipes To Cook</button>
       <form class='search-bar'>
+        <select class="tags-menu">
```
Comment (Billwilke42) [inline src/index.html:19]: Love the functionality of this!

File: src/recipe.js [hunk 4]
```diff
@@ -38,3 +38,58 @@ class Recipe {
...
+/*
+<section class="recipes-header">
+          
+
+
+
+
+
+
+
```
Comment (Billwilke42) [inline src/recipe.js:53]: Well done on this method!!

**Pull request merged**